python/docs/source/getting_started/ps_videos_blogs.rst

```diff
@@ -0,0 +1,130 @@
+======================
+Koalas Talks and Blogs
```

Comment:
Anyway, should we keep this here even if the name of talk or blog posts cannot be changed from Koalas ??

Reply:
I think we should probably remove it. Let's do it separately. This PR ports the docs as is first.